src/test/java/automationpractice/com/pages/AddressPage.java

```diff
@@ -10,9 +10,7 @@
 
 public class AddressPage extends PageObject {
 
-
     private final String DELETE_BUTTON_XPATH_PATTERN = "//*[contains(text(), '%s')]//ancestor::*[contains(@class, 'address') and not(contains(@class, 'addresses'))]//*[contains(@class,'icon-remove')]";
```

Comment:
Ta linijka jest za długa, rozdziel to na dwie linijki. W intelij powinna Ci się wyświetlać po prawej stronie szara pionowa linia. Kod nie powinien jej przekraczać